static/app/components/tabs/tab.tsx

```diff
@@ -101,7 +101,7 @@ export const BaseTab = forwardRef(
           ref={ref}
         >
           <VariantStyledInteractionStateLayer hasSelectedBackground={false} />
-          <VariantFocusLayer />
+          <FilledFocusLayer />
```

Comment:
This is a visual change that was bothering me about the old tab:

Before:
<img width="165" alt="image" src="https://github.com/user-attachments/assets/78ae00c6-19aa-4cc1-9de5-ed8f6540dcbc">

After:
<img width="163" alt="image" src="https://github.com/user-attachments/assets/d83702ef-8815-41f5-b795-bedd034f226a">

I personally like this more, but I think it could go either way. Will check with Vu on what he prefers.